src/Pivot.jl

```diff
@@ -0,0 +1,71 @@
+"""
+unstack_name(df::AbstractDataFrame, rowkeys, colkey, value)
+
+A helper function for unstack to name the columns being unstacked using the column name the values are present in.
+"""
+function unstack_name(df::AbstractDataFrame, rowkeys, colkey, value)
+    unstack(df::AbstractDataFrame, rowkeys, colkey, value, renamecols = x -> Symbol(string(value), "_", x))
+end
+
+
+
+"""
+pivot_wider(df::AbstractDataFrame; names_from =  names_cols, values_from = values_cols)
+
+A function to widen a data frame (e.g. unstack)  but in the frameowrk of dplyr.
```

Comment:
Add a link to the tidyr documentation for pivot_wider?

Reply:
Added the link to dplyr pivot_wider page.